src/tests/mocks/erc721_mocks.cairo

```diff
@@ -114,8 +114,12 @@ mod CamelERC721Mock {
     use openzeppelin::token::erc721::ERC721Component;
     use starknet::ContractAddress;
 
-    component!(path: ERC721Component, storage: erc721, event: ERC721Event);
     component!(path: SRC5Component, storage: src5, event: SRC5Event);
+    component!(path: ERC721Component, storage: erc721, event: ERC721Event);
+
+    // SRC5
+    #[abi(embed_v0)]
+    impl SRC5Impl = SRC5Component::SRC5Impl<ContractState>;
```

Comment:
if we're doing thing i think we should apply it to storage and events as well, and make it consistent across the codebase. for another PR I guess